src/main/java/com/iab/openrtb/request/Geo.java

```diff
@@ -75,12 +83,25 @@
      */
     String city;
 
-    /** Zip or postal code. */
+    /**
+     * Zip or postal code.
+     */
     String zip;
 
-    /** Local time as the number +/- of minutes from UTC. */
+    /**
+     * Local time as the number +/- of minutes from UTC.
+     */
     Integer utcoffset;
 
-    /** Placeholder for exchange-specific extensions to OpenRTB. */
+    /**
+     * Placeholder for exchange-specific extensions to OpenRTB.
+     */
     ObjectNode ext;
+
+    /**
+     * Return true if all fields are not defined.
+     */
+    public boolean isEmpty() {
```

Comment:
While this could be considered extreme, we consistently keep (or strive to) any logic (except object instantiation shortcuts) out of model/proto classes. Can we move this method elsewhere too please?

Additionally, checking fields presence on the object feels more performance friendly. While this implementation may be a bit more robust in case of potential changes to the `Geo` class (especially fields addition), same safety could be achieved with a right unit test.